package.json

```diff
@@ -9,24 +9,25 @@
     "@testing-library/react": "^9.3.2",
```

Comment:
Make sure to pin any dependencies you add to the exact version by deleting the prefixed carrot or using `yarn add --exact`. Pulling in the changes from merge conflicts should fix most of this.